codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/typescript/model/TypeScriptBaseTypes.kt

```diff
@@ -10,8 +10,8 @@ object TypeScriptBaseTypes : LanguageBaseTypes(
     doubleType = nativeTypeScriptType("number"),
     stringType = nativeTypeScriptType("string"),
     booleanType = nativeTypeScriptType("boolean"),
-    dateTimeType = nativeTypeScriptType("string"),
```

Comment:
This one wouldn't work since the string date is interpreted as string so trying to call a date method will induce type error.